gitensite/apps/bookinfo/models.py

```diff
@@ -73,4 +77,16 @@ def metadata(self):
         if not self._pandata:
             self._pandata=Pandata()
             self._pandata.load(self.yaml)
-        return self._pandata.metadata
\ No newline at end of file
+        return self._pandata.metadata
+
+class Cover(models.Model):
+    cover_id = models.IntegerField(unique=True)
+    book_id = models.ForeignKey(Book, on_delete=models.CASCADE, db_index=True)
```

Comment:
this field should be called just book, or else you'll have code like "book_id.id"

Reply:
That's a good point. We will change it